components/Tweet.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-container>
+    <v-row no-wrap no-gutters>
+      <v-col>
+        <v-row>
+          <v-col class="d-flex justify-center align-center">
+            <v-card-title justify-center>
+              <v-avatar color="cyan lighten-2 avatar-border" size="100">
+                <v-img
+                  class="elevation-6"
+                  :alt="tweet.name"
+                  :src="tweet.profile_image_url_original"
+                ></v-img>
+              </v-avatar>
+            </v-card-title>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col class="d-flex justify-center">
+            <v-card-text class="text-body-1 tweet-content" height="400px">
+              {{ tweet.text }}
+            </v-card-text>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col class="d-flex justify-center align-center">
+            <v-card-actions>
+              <v-list-item>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <b> @{{ tweet.username }} </b>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </v-card-actions>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+<script>
+export default {
+  props: {
+    tweet: {
+      required: true,
+      type: Object,
+    },
+  },
+};
+</script>
+<style>
+.avatar-border {
+  border: 3px solid;
```

Comment:
we want to use `rem`s instead of `px` from now on to support a11y and people needing larger fonts and screen zooming

Reply:
While it can make sense to use relative units like `rem` for font sizes specifically, we shouldn't get caught up in the cargo-culting of using it for everything. That's a misconception and a twisting of the original intent of `rem`. Things like borders will make no difference for screen readers / zooming, it's more intuitive to specify them in `px`, and moreover, our UI framework (Vuetify) does this extensively.

As an aside, there's actually no specification in the [a11y checklist](https://www.a11yproject.com/checklist/) for `rem` usage, as all modern browsers will zoom `px` as well, but it could potentially have some use for really old tech, when used for font sizes.

For accessibility, our time is better spent focusing on `aria` labels and `alt` tags.

Reply:
Oops! You're right. Border does not need rem. 

But things related to positioning do matter, such as padding, margin, etc. so the page properly scales with the base font. 

For the design system, we did some investigation and we're probably going to migrate away from vuetify's design system/css styling as it's not very mature. (Note: Not migrating away from vuetify)

Reply:
With that being said though. Rem and px mixed is the worst. It would be better to just use px everywhere. And rem is a pain to calculate / less intuitive. We should bring up this discussion in the slack chat to see if anyone has ideas. 

For now I'd say hold off of this part, but the css classes for 1 liners we definitely want to get rid of. It's an unnecessary overcomplication. 
In case you're wondering, Sass variables are how we can provide consistent sizing / standards across the page.

Reply:
Ok roger. Let's have that discussion. I'm pretty categorically against using rems for border sizes. I think there's a clean enough separation if we specifically use rem for fonts and px elsewhere, but again, we'll take that up elsewhere.